src/reference/01-General-Info/04-Bintray-For-Plugins.md

```diff
@@ -115,6 +111,28 @@ API key for future use.
 *NOTE: We have to do this before we can link our package to the sbt
 org.*
 
+### User Documentation
+
+You should add a few lines to your plugin's README file so users know how to modify their `project/plugins.sbt` in order to use your plugin.
+SBT plugins always use Ivy style patterns, so the resolver incantation is a bit verbose.
+As always, the first component in the resolver is just a human-friendly name for the resolver.
```

Comment:
This is not meant to be a general tutorial on publishing Scala-related packages to BinTray. From the title it is clear that this documentation is specifically focused on issues relating to publishing SBT plugins to BinTray. If there currently is an unmet need for a general tutorial on publishing Scala-related packages to BinTray, that would be a separate issue, handled in another document.